src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageAction.java

```diff
@@ -40,9 +41,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
 
         InstructorFeedbackResultsPageData data = new InstructorFeedbackResultsPageData(account, sessionToken);
         String selectedSection = getRequestParamValue(Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYSECTION);
+        SectionDetail selectedSectionDetail = null;
```

Comment:
What is the meaning of null of SectionDetails?

Reply:
This is to initialise the selectedSectionDetail to be able to be called in the later stage of the code